src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -48,8 +49,15 @@
   public CompilerOptions getCompilerOptions() {
     final CompilerOptions options = new CompilerOptions();
     options.setClosurePass(true);
+
+    // All diagnostics are WARNINGs (or off) and thus ignored unless debug == true.
+    // Only report issues (and fail for them) that are specifically causing problems for Clutz.
+    // The idea is to not do a general sanity check of Closure code, just make sure Clutz works.
+    // Report missing types as errors.
```

Comment:
As discussed offline, projects have different settings, and we just used to treat even every warning as an error. Also, we're really not in the business of telling people whether their Closure code is ok, that's best left to the actual JS binary rule.